### PR TITLE
Programming languages - "Lua" Complete Resources

### LINKED ISSUE #1007: Programming languages - Add complete lua resources
### Idea Contribution

- [X] I have read all the resource request issues.
- [X] I'm interested in working on this issue
- [X] I'm part of GSSOC organization

### Where do you want to add the resource? Specify the category and section.

Adding lua complete resources 

### Provide the URLs of the resources you are adding

_No response_

### Any alternative approaches/features

_No response_

### Additional Context

_No response_

### Code of Conduct

- [X] I follow the [CONTRIBUTING GUIDELINE](https://github.com/jfmartinz/ResourceHub/blob/main/CONTRIBUTING.md) of this project.

### PR DESCRIPTION
**Pull request by aditya-bhaumik**
# Pull Request

## Description
@jfmartinz I have added the resources for Lua programming language please review and merge this pull request  
Closes #1007 

## Type of change
- [x] Adding a new Resource
- [ ] Adding a new Category
- [ ] Documentation enhancement or fixes
- [ ] Website enhancement or fixes

## Checklist:
- [x] I read carefully [CONTRIBUTING.md](https://github.com/jfmartinz/ResourceHub/blob/main/CONTRIBUTING.md)
- [x] I have performed a self-review of my own code
- [x] I have commented my code, particularly in hard-to-understand areas
- [x] I have made corresponding changes to the documentation

### CODE REVIEW

**Comment by jfmartinz** [comment]
Issue number

**Comment by aditya-bhaumik** [comment]
Fixes #1007

**Comment by aditya-bhaumik** [comment]
@jfmartinz please review and merge this pull request

**Comment by aditya-bhaumik** [comment]
> #1007

File: Programming Languages/lua/README.md [hunk 1]
```diff
@@ -0,0 +1,248 @@
...
+  </tr>
+</table>
+
+
+## Projects
+<table width="100%">
+  <tr>
+    <th>Project Name</th>
+    <th>Description</th>
+    <th>Link</th>
```
Comment (jfmartinz) [inline Programming Languages/lua/README.md:210]:
```suggestion

```

File: Programming Languages/lua/README.md [hunk 2]
```diff
@@ -0,0 +1,248 @@
...
+  </tr>
+  <tr>
+    <td>Pong game</td>
+    <td>Making the pong game from scratch using the lua programming language.</td>
+    <td><a href="https://www.angrybirds.com">Pong game</a></td>
+  </tr>
+  <tr>
+    <td>Guess a number game</td>
+    <td>Learn Lua by writing a "guess the number" game</td>
+    <td><a href="https://opensource.com/article/20/12/lua-guess-number-game">Guess a number game</a></td>
```
Comment (jfmartinz) [inline Programming Languages/lua/README.md:220]:
```suggestion

```

File: Programming Languages/lua/README.md [hunk 3]
```diff
@@ -0,0 +1,248 @@
...
+<table width="100%">
+  <tr>
+    <th>Project Name</th>
+    <th>Description</th>
+    <th>Link</th>
+  </tr>
+  <tr>
+    <td>Pong game</td>
+    <td>Making the pong game from scratch using the lua programming language.</td>
+    <td><a href="https://www.angrybirds.com">Pong game</a></td>
```
Comment (jfmartinz) [inline Programming Languages/lua/README.md:215]:
```suggestion

```

**Review by jfmartinz — CHANGES_REQUESTED** [review]
Add the URLs into 'Resource Name'. Remove the third col

**Comment by aditya-bhaumik** [comment]
@jfmartinz i have made the changes requested please merge my pull request

**Comment by aditya-bhaumik** [comment]
@jfmartinz please merge this pull request under gssoc'24

**Review by jfmartinz — CHANGES_REQUESTED** [review]
The URL should be included in the name of the resource, same thing you did in the  web resources, remove the third col
image

**Comment by aditya-bhaumik** [comment]
I already  made the changes  

image

**Review by jfmartinz — APPROVED** [review]
LGTM

**Pull request merged**